lib/cookbook-release/changelog.rb

```diff
@@ -44,6 +85,22 @@ def markdown
       end.join("\n")
     end
 
+    def markdown_priority
+      risky_commits = changelog.select { |c| c.risky? || c.major? }
+      result = if risky_commits.empty?
+                 "*#{NO_RISKY}*\n\n"
```

Comment:
I would not display anything if there are no risky. Otherwise we just clutter the output.